File: lexicon/help.js
module.exports = {
  pattern: 'help',
  reply: function(message, match, reply) {
    reply("Here's what you can ask me: \n- search add-ons ...\n- search author ...\n- search in forums ...\n- search wiki for ...\n- search source for ...\n- search lua for ...\n- live version\n- pts version\n- joke\n- create meme ... ...; ...\n- memes\n- help");
  }
};

File: lexicon/help/help.go
package help

import (
	"github.com/esoui/lexicon/bot"
)

var Expr = `h[ea]lp|(.*)`

func Handler(b *bot.Bot, m bot.Message, match []string) {
	b.Reply(m, "Hey! Here's [what I can do for you](http://wiki.esoui.com/Lexicon).")
}
